modules/Rock.cmake

```diff
@@ -56,6 +56,9 @@ macro (rock_init PROJECT_NAME PROJECT_VERSION)
     rock_use_full_rpath("${CMAKE_INSTALL_PREFIX}/lib")
     include(CheckCXXCompilerFlag)
     include(FindPkgConfig)
+    
+    OPTION(ROCK_USE_CXX11 "Compile package using the C++11 standard" ON)
```

Comment:
:-1: Let's keep it OFF by default and turn it on on systems that support it. I really don't see the downside to that.